### ISSUE TITLE
Configurable middleware

### ISSUE THREAD

**Issue by ernierasta** [adhocteam/pushup#98]
Hi Paul!
First - great work! It is very intriguing way to write web apps!

Do you think it would be possible to have configurable (on project level) middleware? I have [scs.LoadAndSave](https://pkg.go.dev/github.com/alexedwards/scs/v2#SessionManager.LoadAndSave) in mind, but this is general topic.

**Comment by paulsmith** [comment]
Yes, I desire this, too. It needs a design discussion. Probably the easiest thing to do would be a well-known file in the project directory named like `config.go` or something where you are given the configured Pushup app as an argument to a function. If Pushup detects that file it will call that function at some point during initialization of the compiled app.
